lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1715,6 +1737,9 @@ def _resource_sku_to_capability(  # noqa: C901
                         azure_raw_capabilities[
                             location_capability.name
                         ] = location_capability.value
+                node_space.zones = search_space.SetSpace(
```

Comment:
@squirrelsc This is where I set the zones. It is writing to cache fine. Still doesn't show up in the other location though.

Is there a capability merge function or something that I missed?

Reply:
"Standard_D4s_v4": {
            "location": "westus3",
            "vm_size": "Standard_D4s_v4",
            "capability": {
                "type": "requirement",
                "name": "westus3_Standard_D4s_v4",
              ...
                "zones": {
                    "is_allow_set": true,
                    "items": [
                        "1",
                        "3",
                        "2"
                    ]
                },

Reply:
I may not understand correctly. So if it writes to the cache, it should be able to loaded back to the vm size information. For other loctions, do you mean the locations, which you haven't remove the cache? You can delete the whole cache folder to make sure all locations are refreshed.

Reply:
Yes, I deleted all caches. The new cache is correct but it is not loaded into node.capability.zones

![image](https://github.com/microsoft/lisa/assets/24720843/0bb53b11-e820-4f6b-863c-85b9308afcb9)

Reply:
Does it have to do with where this node space is coming from?

``` python
 for node_space in environment.runbook.nodes_requirement:
```

Reply:
Yes, please try if it's in node_space. The node.capability may not be set at this moment.